cmd/insgorund/insgorund.go

```diff
@@ -87,6 +91,29 @@ func main() {
 		os.Exit(1)
 	}
 
+	if *metricsAddress != "" {
+		ctx := context.TODO()
+
+		metricsConfiguration := configuration.Metrics{
+			ListenAddress: *metricsAddress,
+			Namespace:     "insgorund",
+			ZpagesEnabled: true,
+		}
+
+		m, err := metrics.NewMetrics(ctx, metricsConfiguration, metrics.GetInsgorundRegistry())
+		if err != nil {
+			log.Fatal("couldn't setup metrics ", err)
+			os.Exit(1)
+		}
+		err = m.Start(ctx)
+		if err != nil {
+			log.Fatal("couldn't setup metrics ", err)
+			os.Exit(1)
+		}
+
+		defer m.Stop(ctx) // nolint: errcheck
```

Comment:
we should catch SIGTERM and SIGINT for gracefully stop like in insolard. @ruz ?

Reply:
I will create a task

Reply:
@AndreyBronin, comment INS-891 in case you have some insights.